pages/organizations/_organizationId/index.vue

```diff
@@ -0,0 +1,442 @@
+import OrganizationTypeSelector from '~/components/controls/selectors/OrganizationTypeSelector';
+import OrganizationContactSelector from '~/components/controls/selectors/OrganizationContactSelector';
+import CountrySelector from '~/components/controls/selectors/CountrySelector';
+import RepeatedFormInput from '~/components/controls/RepeatedFormInput';
+
+export default {
+  name      : 'EditOrganization',
+  components: {
+    BFormInput,
+    BFormGroup,
+    BForm,
+    BFormTextarea,
+    BFormCheckbox,
+    BFormInvalidFeedback,
+    OrganizationTypeSelector,
+    CountrySelector,
+    RepeatedFormInput,
+    OrganizationContactSelector,
+  },
+  data() {
+    return {
+      organizationId: this.$route.params.organizationId || '',
+      organization  : {},
+    };
+  },
+  computed: {
+    organizationNameValid() { return (this.organization.name || '').trim() !== ''; },
+    governmentValid() { return (this.governmentDisabled ? null : ((this.organization.government || '').trim() !== '')); },
+    organizationTypeValid() { return !this.organization.organizationTypeId; },
+    countryValid() { return !this.organization.country; },
+    governmentDisabled: {
+      get() {
+        const selectedOrgType = this.orgTypes
+          .find((type) => type.organizationTypeId === this.organization.organizationTypeId);
+        return !(selectedOrgType && selectedOrgType.acronym === 'GOV');
+      },
+    },
+    ...mapGetters({ orgTypes: 'organizations/types' }),
+  },
+  mounted: load,
+  methods: {
+    save,
+    close,
+    refreshUrl,
+  },
+};
+
+function initialOrganization() {
+  return  {
+    organizationId       : '',
+    name                 : '',
+    acronym              : '',
+    organizationTypeId   : '',
+    focalPointContactId  : '',
+    headContactId        : '',
+    government           : '',
+    address              : '',
+    city                 : '',
+    state                : '',
+    country              : '',
+    postalCode           : '',
+    phones               : [],
+    faxes                : [],
+    emails               : [],
+    emailCcs             : [],
+    webs                 : [],
+    notes                : '',
+    isValidated          : false,
+    isDateOfBirthRequired: false,
+    isHidden             : false,
+    isInMailingList      : false,
+  };
+}
+
+async function load() {
+  try {
+    const { organizationId } = this;
+
+    this.organization = organizationId === 'new'
+      ? initialOrganization()
+      : await this.$kronosApi.getOrganization(this.organizationId);
+  } catch (error) {
+    // give Good alert of incorrect id
+
+    this.$toast.error(error.messsage || `TODO: error ${error}`);
+  }
+}
+
+async function close() {
+  this.$router.go(-1);
+}
+
+async function save(event) {
+  try {
+    const form = event.currentTarget;
+
+    if (!form.checkValidity()) return;
+
+    const { organization, organizationId } = this;
+
+    const result = organizationId === 'new'
```

Comment:
`organization.organizationId` will be either `''` or `'00787787AF...'`. Bases on the new organization mock object `new` is only on the route paramaters but on the object (see line 374 https://github.com/scbd/kronos-web/pull/59/files#diff-b362c8b50b85882d46b34ffddf0023f8941d92dc78938e8f502f8c57876881b7R374)

better to test empty value instead

```suggestion
    const result = !organizationId
```

Reply:
DONE! changes done as suggested.